doc/run_on_gcp.md

```diff
@@ -0,0 +1,169 @@
+    ```
+
+## Set Up Google CloudSQL Instance
+
+1. Create a VPC (Virtual Private Cloud) on Google Cloud Platform. [VPC](https://cloud.google.com/vpc/docs/overview)
+   provides a logically isolated network for your GKE clusters to communicate
+   with services like CloudSQL, so traffic between SQLFlow server and CloudSQL
+   instance does not go through public Internet.
+    ``` bash
+    > gcloud compute networks create sqlflow-demo-vpc --subnet-mode custom
+    ```
+
+1. Reserve an ip range for the created VPC. This ip range will be used for private service
+   connections from applications (in our case, SQLFlow server) running within the created VPC
+   to communicate with CloudSQL instance.
+    ``` bash
+    > gcloud beta compute addresses create vpc-peering-sqlflow-demo-vpc --global --purpose VPC_PEERING --description="For sqlflow with cloudsql private connection" --addresses 10.20.0.0 --prefix-length 16 --network sqlflow-demo-vpc
+    ```
+
+1. Create a private service connection between the created VPC and Google API services.
+    ``` bash
+    > gcloud services vpc-peerings connect --service servicenetworking.googleapis.com --ranges vpc-peering-sqlflow-demo-vpc --network sqlflow-demo-vpc
+    ```
+
+    You can read more about private services access [here](https://cloud.google.com/vpc/docs/configure-private-services-access).
+    To summary, this private connection establishes a VPC Network Peering
+    connection between our VPC (in which our application will be running), and
+    the service producer's VPC network (the CloudSQL instance we will create
+    will be running within a different VPC network managed by GCP internally).
+
+1. Create a CloudSQL MySQL instance.
+    ``` bash
+    > gcloud beta sql instances create sqlflow-cloudsql-instance --network sqlflow-demo-vpc --no-assign-ip --tier db-n1-standard-2 --region us-central1
+    Creating Cloud SQL instance...done.
+    NAME                       DATABASE_VERSION  LOCATION       TIER              PRIMARY_ADDRESS  PRIVATE_ADDRESS  STATUS
+    sqlflow-cloudsql-instance  MYSQL_5_7         us-central1-a  db-n1-standard-2  -                10.20.0.3        RUNNABLE
+    ```
+
+    You can find the *Instance connection name* and *Private IP address* for the
+    newly created CloudSQL instance at the [CloudSQL console](https://console.cloud.google.com/sql/instances/sqlflow-cloudsql-instance/overview).
+    Note that
+    - the private ip address is within the IP range we reserved in the previous step.
+    - You should be able to see a private connection between your VPC and the
+      newly created CloudSQL instance on the VPC console.
+
+1. Configure the default CloudSQL MySQL user account password
+   For this demo purpose, we assume the *root* user of our MySQL instance has
+   password *root*. To configure this, run the following:
+    ``` bash
+    > gcloud sql users set-password root --host=% --instance=sqlflow-cloudsql-instance --password=root
```

Comment:
I haven't tested this command, does `--host=%` works well?

Reply:
Yes, host=% denotes unrestricted host name

https://cloud.google.com/sdk/gcloud/reference/sql/users/set-password#--host